### PR TITLE
Fix offshore wind capacity by adding it as offwind-AC

### PR DESCRIPTION
**Pull request by toniseibold**
Closes https://github.com/PyPSA/pypsa-eur/issues/1088

## Changes proposed in this Pull Request
Adding offshore capacity from IRENA as `offshore-ac`.

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [x] Changed dependencies are added to `envs/environment.yaml`.
- [x] Changes in configuration options are added in all of `config.default.yaml`.
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv`.
- [x] A release note `doc/release_notes.rst` is added.

### CODE REVIEW

File: scripts/add_existing_baseyear.py [hunk 1]
```diff
@@ -254,7 +258,11 @@ def add_power_capacities_installed_before_baseyear(n, grouping_years, costs, bas
         name_suffix = f" {generator}{suffix}-{grouping_year}"
         name_suffix_by = f" {generator}{suffix}-{baseyear}"
         asset_i = capacity.index + name_suffix
-        if generator in ["solar", "onwind", "offwind"]:
+        if generator in ["solar", "onwind", "offwind-ac"]:
+            if generator == "offwind-ac":
+                cost_key = "offwind"
+            else:
+                cost_key = generator
```
Comment (fneum) [inline scripts/add_existing_baseyear.py:265]:
```suggestion
        cost_key = generators.split("-")[0]
```
This should also work and is simpler.

File: scripts/add_existing_baseyear.py [hunk 2]
```diff
@@ -109,12 +109,16 @@ def add_existing_renewables(df_agg, costs):
                 name = f"{node}-{carrier}-{year}"
                 capacity = nodal_df.loc[node, year]
                 if capacity > 0.0:
+                    if carrier == "offwind-ac":
+                        cost_key = "offwind"
+                    else:
+                        cost_key = carrier
```
Comment (fneum) [inline scripts/add_existing_baseyear.py:115]:
```suggestion
                    cost_key = carrier.split("-")[0]
```

**Review by fneum — CHANGES_REQUESTED** [review]
LGTM, just double-check the `.split()` suggestion and merge at will.

File: scripts/add_existing_baseyear.py (hunk 1 above)
Comment (toniseibold) [reply scripts/add_existing_baseyear.py:262]: I need the line `if generator in ["solar", "onwind", "offwind-ac"]` to iterate over renewables only.

**Pull request merged**